.github/workflows/build.yml

```diff
@@ -50,8 +53,8 @@ jobs:
           file: ./build/docker/agent/Dockerfile
           load: true
           tags: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME}}:test
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
+          cache-from: type=local,dest=/cache/
+          cache-to: type=local,src=/cache/,mode=max
```

Comment:
Have you considered that the local cache will not be deleted automatically?

https://docs.docker.com/build/ci/github-actions/cache/#local-cache

Reply:
From what I have seen the cache keeps growing, so automatic deletion should be added.
However if it keeps growing do we even need it if a new image is added to the cache on every build?